_articles/tr/code-of-conduct.md

```diff
@@ -7,9 +7,9 @@ toc:
   why-do-i-need-a-code-of-conduct: Neden bir davranış kural listesine ihtiyacım var?
   establishing-a-code-of-conduct: Davranış kuralları oluşturmak
   deciding-how-youll-enforce-your-code-of-conduct: Davranış kurallarınızı nasıl uygulayacağınıza karar verme
-  enforcing-your-code-of-conduct: Davranış kurallarınızı güçlendirmek
+  enforcing-your-code-of-conduct: Davranış kurallarınızı uygulama
   your-responsibilities-as-a-maintainer: Bir koruyucu olarak sorumluluklarınız
-order: 8
```

Comment:
hi, I think we need these to stay as integers in order for the frontmatter to be read correctly!

Reply:
Hi @deniseyu ,

Many thanks for the review.

I don't know why they were deleted. Probably, the translation tool that I am using caused this. I will check all the files too and fix the same issues if there are any.

Reply:
no problem! I also share your suspicion that the tool did it! Let me know if you need any help with scoping down this PR to only contain the awesome translations!

Reply:
Hi @deniseyu ,

I think I have fixed all the issues in Turkish translation files. But, the test fails in a Chinese file because of the failure of an external url.

Reply:
Thanks @umutphp for fixing this! I'm looking into that failure right now, and will take care of it, no further action needed from you. Thanks for your patience!